pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+**Description of changes:**
+
+
+
+**Checklist before we merge your thing:**
+- [ ] The author is ready to merge
+- [ ] All changes are described above
+- [ ] 24 hours for comment have passed, or this is a bugfix
+- [ ] Hosted and tested; it compiles and works
```

Comment:
Split "compile" and "tested" into two different checks.

Reply:
If it was tested then it compiles.

Reply:
Sure, but I compile without testing on occasion.

Reply:
Then it's not Hosted and Tested ^.-

Reply:
Bahh!